### DISCUSSION TITLE
UI enhancements?

### DISCUSSION THREAD

**Discussion by ggjiza** [seerge/g-helper#1031] [Ideas]
screenshot: (https://github.com/seerge/g-helper/assets/133283942/66483cc0-3a00-49a4-be96-f2d79586a7d6)
For laptops that don't have anime matrix, could the "laptop screen" buttons be expanded to fill the bar?
Also if the "laptop screen" is turned off, could that bar be hidden? it also shows "120Hz+ OD" which is normally 165.

**Comment by seerge** [comment]
It's intentionally not hidden, to show that there is such setting, or people will start asking why it's gone.

As for showing 120 instead of 165, it will remember your max as soon as it's able to communicate to screen when it's on.

**Reply by ggjiza** [reply]
I understand, But it's will look much cleaner if the text remains "Laptop screen turned off" while the buttons are hidden.
Side note, if the "anime matrix"/4th empty button space can be filled with a custom button for some functions like (project/display settings), RAM/CPU/GPU usage for example, that would be awesome.
